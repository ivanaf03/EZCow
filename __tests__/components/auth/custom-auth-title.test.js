import React from "react";

import { render } from "@testing-library/react-native";

import CustomAuthTitle from "../../../components/auth/custom-auth-title";

describe("CustomAuthTitle", () => {
  it("renders correctly", () => {
    const tree = render(<CustomAuthTitle text="Test" />);
    expect(tree).toMatchSnapshot();
  });
});