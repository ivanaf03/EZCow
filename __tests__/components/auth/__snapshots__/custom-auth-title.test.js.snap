// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomAuthTitle renders correctly 1`] = `
<Text
  style={
    [
      {
        "marginBottom": 32,
        "marginTop": 32,
      },
      {
        "paddingBottom": 16,
        "paddingLeft": 16,
        "paddingRight": 16,
        "paddingTop": 16,
      },
      {
        "textAlign": "center",
      },
      {
        "color": "#ffffff",
      },
      {
        "fontSize": 48,
        "lineHeight": 48,
      },
      {
        "fontFamily": "Nunito-Bold",
      },
    ]
  }
>
  Test
</Text>
`;
